usr/share/rear/setup/default/010_pre_recovery_script.sh

```diff
@@ -1,4 +1,11 @@
 if test "$PRE_RECOVERY_SCRIPT" ; then
-	Log "Running PRE_RECOVERY_SCRIPT '${PRE_RECOVERY_SCRIPT[@]}'"
-	eval "${PRE_RECOVERY_SCRIPT[@]}"
+    Log "Running PRE_RECOVERY_SCRIPT '${PRE_RECOVERY_SCRIPT[@]}'"
+    eval "${PRE_RECOVERY_SCRIPT[@]}"
```

Comment:
cosmetic remark: I think it is better to separate formatting changes (tab expansion) into a separate commit (can be in the same PR).

Reply:
OK